go/teams/seitan.go

```diff
@@ -101,7 +108,7 @@ func ParseSeitanTokenFromPaste(token string) (string, bool) {
 		}
 		return token, true
 	}
-	if IsSeitany(token) {
+	if IsSeitany(token) || IsSeitanyV2(token) {
```

Comment:
as discussed yestyerday, maybe we can get away with having just `IsSeitany` that covers both

Reply:
done